From 72340ebaa98154b3b5ff7b5b7e4d2ddc62b2c8db Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Wed, 9 Oct 2019 17:47:46 +0200
Subject: [PATCH] procps-ng: fix selection of all packages to none

Setting it to explicit default no causes that when we want to compile
all packages that they won't be compiled. This means that no procps-ng
package is compiled in packages repository.

Default behavior is to not select so instead of being clever just let
other decide like for any other package.
---
 utils/procps-ng/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/utils/procps-ng/Makefile b/utils/procps-ng/Makefile
index 11c69426f..3d43b44f0 100644
--- a/utils/procps-ng/Makefile
+++ b/utils/procps-ng/Makefile
@@ -73,7 +73,6 @@ define GenPlugin
    $(call Package/procps-ng/Default)
    DEPENDS:=procps-ng
    TITLE:=Applet $(2) from the procps-ng package
-   DEFAULT:=n
    ALTERNATIVES:=200:$(3)/$(2):$(3)/$(1)
  endef
 
-- 
2.23.0

